package.json

```diff
@@ -7,18 +7,13 @@
 		"test": "test"
 	},
 	"scripts": {
-		"test": "npm run settings && npm run test-default && npm run test-full && npm run test-empty && npm run test-output && node test/test.js",
-		"test-functions": "node-sass test/scss/index.scss test/css/functions.css",
-		"test-empty": "node-sass test/scss/empty.scss test/css/empty.css",
-		"test-default": "node-sass test/scss/default.scss test/css/default.css",
-		"test-full": "node-sass test/scss/full.scss test/css/full.css",
-		"test-output": "node-sass test/scss/output.scss test/css/output.css",
+		"test": "npm run settings && node test/node-sass.js && node test/sass.js && node test/test.js",
 		"settings": "node scripts/json-to-css.js",
 		"dev": "npm run settings",
 		"build": "npm run settings && rm -rf dist/**/* && npm run build-dist",
 		"build-dist": "npm run build-default && npm run build-full",
-		"build-default": "node-sass dist-input/default.scss dist/henris.css --output-style nested && node-sass dist-input/default.scss dist/henris.min.css  --output-style compressed",
-		"build-full": "node-sass dist-input/full.scss dist/henris.full.css nested --output-style && node-sass dist-input/full.scss dist/henris.full.min.css  --output-style compressed",
+		"build-default": "sass dist-input/default.scss dist/henris.css --output-style nested && sass dist-input/default.scss dist/henris.min.css  --output-style compressed",
+		"build-full": "sass dist-input/full.scss dist/henris.full.css nested --output-style && sass dist-input/full.scss dist/henris.full.min.css  --output-style compressed",
 		"prepublishOnly": "npm run test && npm run build",
```

Comment:
so every script uses - as separator, except this one which uses camelcase?

Reply:
Ehh.. jeps, maar die prepublishOnly moet zo van npm? Die wordt uitgevoerd voor publish.

Reply:
Maybe change all ohter scripts to same format (camelcase). It's a JS script so it should have a JS name

Reply:
Its not, these are cli commands, therefore camelCase is weird.

Reply:
that solves the issue as well

Reply:
Will check, delete and take to next release.

Reply:
Or maybe not.. maybe we are still in this transitional fase...
its not really clear in the docs